src/worldstate/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(worldstate)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  std_msgs
+  message_generation
+)
+
+add_definitions(-std=c++14)
+
+include_directories(
+    ${catkin_INCLUDE_DIRS}
+)
+
+################################################
+## Declare ROS messages, services and actions ##
+################################################
+
+## Generate messages in the 'msg' folder
+add_message_files(
+  FILES
+  state_msg.msg
+)
+
+## Generate added messages and services with any dependencies listed here
+generate_messages(
+  DEPENDENCIES
+  std_msgs
+)
+
+###################################
+## catkin specific configuration ##
+###################################
+
+catkin_package(
+   LIBRARIES worldstate
```

Comment:
We shouldn't be exporting a library here. As we discovered today, this can cause issues when other packages try to use this one. For more info on `catkin_package`, see http://wiki.ros.org/catkin/CMakeLists.txt#catkin_package.28.29

Reply:
Removed the "LIBRARIES worldstate" line!